server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/api.ts

```diff
@@ -129,6 +135,20 @@ export function create(
 		},
 	);
 
+	router.post("/broadcast-signal", (req, res) => {
```

Comment:
As is, this will leave us vulnerable to unauthorized users broadcasting signals. This route needs the appropriate auth and validation middlewares added to it. The following should almost suffice, with the exception of `verifyStorageToken`:

```ts

router.post(
        "/broadcast-signal",
		validateRequestParams("tenantId", "id"),
		throttle(generalTenantThrottler, winston, tenantThrottleOptions),
        // this middleware relies on tenantId and documentId coming from route (/:tenantId/:id)
		verifyStorageToken(tenantManager, config, defaultTokenValidationOptions),
        (req, res) => {
```

See other comment about docId/tenantId validation